join-chat-to-teams-meeting/Chat Experience with Inline Image/README.md

```diff
@@ -14,15 +14,15 @@ products:
 
 This code sample demonstrates how you can support Teams inline image interoperability with the Azure Chat SDK for JavaScript  
 
-This sample includes the code that is explained as part of [this tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/chat-interop/meeting-interop-features-inline-image), which includes a step by step explanation on how to enable inline image support. 
-
+This sample includes the code that is explained as part of [this tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/chat-interop/meeting-interop-features-inline-image), which includes a step by step explanation on how to enable inline image support.
 
 ## Prerequisites
+
 - An active Communication Services resource. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource).
-- A Working [Communication Services chat web app](https://docs.microsoft.com/azure/communication-services/quickstarts/chat/get-started?pivots=programming-language-javascript). 
+- A Working [Communication Services chat web app](https://docs.microsoft.com/azure/communication-services/quickstarts/chat/get-started?pivots=programming-language-javascript).
 - A Teams meeting link that was set up by a Teams business account.
 - A clear understanding from previous QuickStart - [Join your chat app to a Teams meeting](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/chat/meeting-interop?pivots=platform-web).
-- Chat SDK for JavaScript ([@azure/communication-chat](https://www.npmjs.com/package/@azure/communication-chat)) 1.3.2-beta.1 or latest
+- Chat SDK for JavaScript ([@azure/communication-chat](https://www.npmjs.com/package/@azure/communication-chat)) 1.4.0 or greater
```

Comment:
```suggestion
- Chat SDK for JavaScript ([@azure/communication-chat](https://www.npmjs.com/package/@azure/communication-chat)) 
```
IMO we drop the version. Developers can just go to the npm link provided and see the version. This is just less overhead for us to maintain

Reply:
And now that it's GA, you're right - we can.